docs/_source/guides/llms/practical_guides/update_dataset.md

```diff
@@ -0,0 +1,107 @@
+
+::::{tab-set}
+
+:::{tab-item} Argilla 1.14.0 or higher
+```python
+# load the dataset
+dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
+# list of Feedback records to add
+new_records = [...]
+# add records to the dataset
+dataset.add_records(new_records)
+```
+:::
+
+:::{tab-item} Lower than Argilla 1.14.0
+```python
+# load the dataset
+dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
+# list of Feedback records to add
+new_records = [...]
+# add records to the dataset
+dataset.add_records(new_records)
+# push the dataset to Argilla
+dataset.push_to_argilla()
+```
+:::
+::::
+
+To learn about the format that these records follow, check [this page](create_dataset.md#add-records) or go to our [cheatsheet](../../../getting_started/cheatsheet.md#create-records).
+
+## Delete existing records
+
+From `v1.14.0` it is also possible to delete records from a Feedback dataset using the `delete` method like so:
+
+```python
+# load the dataset
+dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
+# delete a specific record
+dataset.records[0].delete()
+```
+
+If you prefer to delete a list of records, you can also achive this in this way:
+
+```python
+# load the dataset
+dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
+# delete a list of records from a dataset
+dataset.delete_records([record for record in dataset.records[:5]])
+```
+
+## Add suggestions to existing records
+
+You can also add suggestions to records that have been already pushed to Argilla:
+
+::::{tab-set}
+
+:::{tab-item} Argilla 1.14.0 or higher
+```python
+# load the dataset
+dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
+# loop through the records and add suggestions
+for record in dataset.records:
+    record.update(suggestions=[...])
+```
+:::
+
+:::{tab-item} Lower than Argilla 1.14.0
+```python
+# load the dataset
+dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
+# loop through the records and add suggestions
+for record in dataset.records:
+    record.set_suggestions([...])
+```
+:::
+::::
+
+To learn about the schema that these suggestions should follow check [this page](create_dataset.md#add-suggestions).
+
+## Update suggestions in existing records
+
+Existing suggestions can also be modified and updated from Argilla `v1.14.0` using the same method as the one in the section above. This will overwrite any existing suggestions.
+
+```python
+# load the dataset
+dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
+# loop through the records and update the suggestions
+for record in dataset.records:
+    record.update(suggestions=[...])
+```
+
+To learn about the schema that these suggestions should follow check [this page](create_dataset.md#add-suggestions).
```

Comment:
Indeed this section can be removed and mentioned as part of the `Add suggestions to existing records` section renaming it to `Add/Update suggestions in existing records`, as the code is exactly the same.

Reply:
If you prefer to keep it, then you should also add the `tab-set` with the different versions of the code-snippet

Reply:
Yes, that's what I thought initially, but I wanted to do it simple like this first and check with you that was the case. I'll do that then 😃